ipykernel/iostream.py

```diff
@@ -215,7 +215,7 @@ async def _handle_pipe_msg(self, msg=None):
         if not self._pipe_flag or not self._is_master_process():
             return
         if msg[0] != self._pipe_uuid:
-            print("Bad pipe message: %s", msg, file=sys.__stderr__)
+            # print("Bad pipe message: %s", msg, file=sys.__stderr__)
```

Comment:
These messages do indicate a problem, and should still be logged, not entirely suppressed. The problem comes from the change  that `sys.__stderr__` is now duplicated and forwarded to users, when this message is meant to only. What we really want to write to is `sys.stderr._original_stdstream_copy`.

I see two options:

1. when we capture the low-level `__stderr__`, we could replace `sys.__stderr__` with the new captured fd, so statements like this do what we originally meant them to (only write to stderr, not captured and forwarded), or
2. keep capturing `__stderr__` and explicitly write to the "truly really actually only original stderr" FD for messages like these.